translations.json

```diff
@@ -0,0 +1,86 @@
+{
+  "de": {
+    "time_min": "%d min",
+    "time_hour_min": "%d std %02d min",
+    "time_hour": "%d std",
+    "from": "von",
+    "at": "an",
+    "now": "jetzt",
+    "in_min": "in %s",
+    "next_min": "und in %s",
+    "next_after_min": "und dananch in %s",
+    "next_timestamp": "und wieder %s",
+    "in_the_past": "In der Vergangenheit",
+    "arrival_from": "Ankunft von %s",
+    "operator_name": "Ein Angebot der %s",
+    "platform_types": {
```

Comment:
Please note that some providers (looking at you, RMV) also have empty platform types (so we should handle `""` and `None`)

Reply:
There are now checks for empty platform types.